### PR TITLE
Service Insights by terraform

### PR DESCRIPTION
**Pull request by lm-madhvi**
Created a functionality in provider for  adding a  service and  logicmonitor datasource for that service .

### CODE REVIEW

File: logicmonitor/provider.go [hunk 1]
```diff
@@ -67,9 +69,9 @@ func Provider() *schema.Provider {
 func providerConfigure(ctx context.Context, d *schema.ResourceData) (interface{}, diag.Diagnostics) {
 	var diags diag.Diagnostics
 
-	id := d.Get("api_id").(string)
-	key := d.Get("api_key").(string)
-	company := d.Get("company").(string) + ".logicmonitor.com"
+ 	id := d.Get("api_id").(string)
```
Comment (lmswagatadutta) [inline logicmonitor/provider.go]: Please remove all the whitespace changes from this file.

File: resources_docs/README.md
Comment (lmswagatadutta) [inline resources_docs/README.md:1]: IMO, this file should be available in the Terraform registry as a document.

**Review by lmswagatadutta — CHANGES_REQUESTED** [review]

File: logicmonitor/provider.go (hunk 1 above)
Comment (lm-madhvi) [reply logicmonitor/provider.go]: Updated the pull request.

File: resources_docs/README.md
Comment (lm-madhvi) [reply resources_docs/README.md:1]: Added.

**Review by lmswagatadutta — APPROVED** [review]

**Pull request merged**